pygfx/materials/_base.py

```diff
@@ -116,4 +118,17 @@ def clipping_planes(self, planes):
             self.uniform_buffer.data["clipping_planes"][i] = planes2[i]
         self.uniform_buffer.update_range(0, 1)
 
-    # todo: clip_intersection
+    @property
+    def clip_intersection(self):
+        """Changes the behavior of clipping planes so that only their
+        intersection is clipped, rather than their union. Default is false.
```

Comment:
Isn't it the other way around? Any matches union right?

Reply:
If the intersection is clipped, you clip less fragments - the union is visible. And the other way around. Could use help rephrasing this :/

Reply:
So I mean that the docstring is technically correct (also see [threejs example](https://threejs.org/examples/?q=clipp#webgl_clipping_intersection), but it can be made more clear. Maybe it needs something like "clipped fragments are not clipped" in there.

Reply:
The thing that confused me was the use of the word "satisfies". It's not immediately clear what that means. How about:

> By default a fragment is visible if it is clipped by any clipping plane. With this set to True, it is visible only if it is clipped by all of the clipping planes.

Reply:
I like how removing the words from set theory make the docstring more clear. I'm wondering whether we can find a better name for the property itself (at the cost of deviating from the ThreeJS name). E.g. `clip_any = True/False` or `clip_mode = "any", "all"`.

Reply:
I really like clip_mode=any/all!